src/main/java/com/ibm/watson/prometheus/NettyServer.java

```diff
@@ -89,6 +89,7 @@ private Channel startServer(boolean tls) throws Exception {
         logger.info("Starting prometheus " + (tls ? "HTTPS" : "HTTP") + " server on port " + port);
         final SslContext sslContext;
         if (tls) {
+            // Note this requires bouncycastle library for java versions >= 15
```

Comment:
Is this a dependency that will need to be added in https://github.com/kserve/modelmesh/pull/33 then?